modules/schematics/src/action/schema.json

```diff
@@ -13,6 +13,15 @@
       },
       "x-prompt": "What should be the name of the action?"
     },
+    "prefix": {
```

Comment:
Does this work? The `name` property already uses the unnamed argument, I think the default of this should be `load`, or the index should be 1 (but I prefer the default option)

Reply:
Yes, this works, but the default option is a better idea